com.unity.cinemachine/Documentation~/CinemachineInputAxisController.md

```diff
@@ -28,3 +31,142 @@ If you like, you can also use CinemachineInputAxisController with your own scrip
 | __Accel Time__ | The time it takes for the input value to accelerate to a larger value |
 | __Decel Time__ | The time it takes for the input value to decelerate to a smaller value |
 
+## Creating your own Input Axis Controller
+
+The default implementation of `CinemachineInputAxisController` can process input from the Input package and from Unity's legacy input system. In the case where you want your input to come from an other source, you may need to create your own input controller.
```

Comment:
What happened to "This example shows how to control a Camera with a slider. It can be easily modified to be used with other objects." ?  That was good, I miss it.

Reply:
Readed